src/world.rs

```diff
@@ -774,6 +775,41 @@ impl World {
         self.archetypes_inner().iter()
     }
 
+    /// Faster way of accessing a Component many times
+    pub fn column<T: Component>(&self) -> Column<'_, T> {
+        let archetypes = self.archetypes.archetypes.as_slice();
+        let entities = self.entities.meta.as_slice();
+        let mut archetype_column_offsets: Vec<Option<usize>> = Vec::new();
+        for i in archetypes.iter() {
+            archetype_column_offsets.push(i.get_state::<T>());
+        }
+
+        Column::new(entities, archetypes, archetype_column_offsets, PhantomData)
+    }
+
+    /// Faster way of uniquely accessing a Component type
+    ///
+    /// It's useful for applications that do large amount of random
+    /// access to same Component type
+    ///
+    /// #Example
+    /// ```
+    /// use::hecs::*;
+    /// let mut world = World::new();
+    /// let ent = world.spawn((123,"abc"));
+    /// let column_mut = world.column_mut::<i32>();
+    /// assert_eq!(*column_mut.get(ent).unwrap(),123);
```

Comment:
It's important to use good style in examples to maximize readability and teach good habits.
```suggestion
    /// let ent = world.spawn((123, "abc"));
    /// let column_mut = world.column_mut::<i32>();
    /// assert_eq!(*column_mut.get(ent).unwrap(), 123);
```